client/src/components/Workflow/WorkflowList.vue

```diff
@@ -143,47 +156,44 @@ export default {
             loading: true,
             message: null,
             messageVariant: null,
-            nWorkflows: 0,
-            workflows: [],
             titleSearchWorkflows: _l("Search Workflows"),
             titleCreate: _l("Create"),
             titleImport: _l("Import"),
             titleRunWorkflow: _l("Run workflow"),
+            workflowItemsModel: [],
+            workflowItems: [],
+            perPage: this.rowsPerPage(20),
```

Comment:
```suggestion
            perPage: this.rowsPerPage(50),
```
20 is kind of low, 50 is what the invocations, review cleanup,  and dataset search components use. (And it would be really cool if we could make this a sticky user preference at one point in the future).

Reply:
I don't agree but I've made the switch in #13725. I think the workflow list is more like a Google search result. I'm imaging a future with rich text descriptions and workflow icons and box-y instead of grid display (ala discussed in the context of simplified workflow interface and seen on say https://nf-co.re/pipelines). In that context - Google's 20 feels right. Invocations/jobs/datasets are more like proper rows in a large table to me and the limit of 50 feels right there.

I think the simplified workflow UI placing the workflows above the invocations also influenced me in thinking they should not consume so much space... but I will admit all of those are future concerns.

Reply:
Rebased into this PR.